package/MDAnalysis/core/topologyattrs.py

```diff
@@ -204,33 +235,32 @@ def _build_stub(method_name, method, attribute_name):
     -------
     The stub.
     """
+
     def stub_method(self, *args, **kwargs):
         message = (
-            '{class_name}.{method_name}() '
-            'not available; this requires {attribute_name}'
+            "{class_name}.{method_name}() "
+            "not available; this requires {attribute_name}"
         ).format(
             class_name=self.__class__.__name__,
             method_name=method_name,
             attribute_name=attribute_name,
         )
         raise NoDataError(message)
 
-    annotation = textwrap.dedent("""\
+    annotation = textwrap.dedent(
+        """\
         .. note::
 
           This requires the underlying topology to have {}. Otherwise, a
           :exc:`~MDAnalysis.exceptions.NoDataError` is raised.
 
 
-    """.format(attribute_name))
-    # The first line of the original docstring is not indented, but the
-    # subsequent lines are. We want to dedent the whole docstring.
-    first_line, other_lines = method.__doc__.split('\n', 1)
-    stub_method.__doc__ = (
-        first_line + '\n'
-        + textwrap.dedent(other_lines)
-        + '\n\n' + annotation
+    """.format(
+            attribute_name
+        )
     )
+    # The original docstring is arrumed to be formatted with black
```

Comment:
Make clear what the black-formatted doc strings look like (and fixed typo)
```suggestion
    # The original docstring is assumed to be formatted with black
    # (i.e., *all text* lines are already indented, the first line 
    # is a triple quote followed by newline)
```

Reply:
Thanks @orbeckst, much better! I agree it was brittle, and I think having automatic formatting here makes it less so (so a win in my book).